en/integrations/plugin-packs/procedures/hardware-servers-dell-omem-snmp.md

```diff
@@ -0,0 +1,170 @@
+| psu status                                | Current status of the power supply                             |       |
+| chassis#hardware.temperature.celsius      | Ambient temperature reading for the chassis front panel module | C     |
+
+<!--Uptime-->
+
+| Metric name           | Description        | Unit  |
+| :-------------------- | :----------------- | :---- |
+| system.uptime.seconds | System uptime      | s     |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prerequisites
+
+### SNMP Configuration
+
+To use this pack, the SNMP service must be properly configured on your device.
+
+### Network flow
+
+The target equipment must be reachable from the Centreon Poller on the UDP/161 SNMP
+port.
+
+## Setup
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Online License-->
+
+1. Install the Centreon Plugin package on every Centreon poller expected to monitor **Dell OME–Modular SNMP** resources:
+
+```bash
+yum install centreon-plugin-Hardware-Servers-Dell-Omem-Snmp
+```
+
+2. On the Centreon Web interface, install the **Dell OME–Modular SNMP** Centreon Pack on the **Configuration > Plugin Packs** page.
+
+<!--Offline License-->
+
+1. Install the Centreon Plugin package on every Centreon poller expected to monitor **Dell OME–Modular SNMP** resources:
+
+```bash
+yum install centreon-plugin-Hardware-Servers-Dell-Omem-Snmp
+```
+
+2. Install the **Dell OME–Modular SNMP** Centreon Pack RPM on the Centreon Central server:
+
+ ```bash
+yum install centreon-pack-hardware-servers-dell-omem-snmp
+```
+
+3. On the Centreon Web interface, install the **Dell OME–Modular SNMP** Centreon Pack on the **Configuration > Plugin Packs** page.
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Configuration
+
+### Host
+
+* Log into Centreon and add a new Host through **Configuration > Hosts**
+* Fill the **Name**, **Alias** & **IP Address / DNS** fields according to your **Dell OME–Modular SNMP** server settings
+* Select the **HW-Server-Dell-OMEM-SNMP-custom** template to apply to the Host
+
+If you are using SNMP Version 3, use the SNMPEXTRAOPTIONS Macro to configure
+your own SNMPv3 credentials combo.
+
+| Mandatory | Name             | Description                                              |
+|:----------|:-----------------|:---------------------------------------------------------|
+|           | SNMPEXTRAOPTIONS | (Default: 'Configure your own SNMPv3 credentials combo') |
+
+## How to check in the CLI that the configuration is OK and what are the main options for? 
+
+Once the plugin is installed, log into your Centreon Poller CLI using the 
+**centreon-engine** user account and test the Plugin by running the following 
+command:
+
+```bash
+/usr/lib/centreon/plugins/centreon_dell_omem_snmp.pl \
+    --plugin=hardware::server::dell::omem::snmp::plugin \
+    --mode=hardware \
+    --hostname='10.0.0.1' \
+    --snmp-version='2c' \
+    --snmp-community='my-snmp-community' \
```

Comment:
some threshold could be great, even reproducing the default it might help to understand the threshold-overload syntax

Reply:
we'lle manage the threshold-overload options differently.